docker/perception/lidar_object_detection/lidar_object_detection.Dockerfile

```diff
@@ -7,20 +7,70 @@ WORKDIR ${AMENT_WS}/src
 
 # Copy in source code 
 COPY src/perception/lidar_object_detection lidar_object_detection
-COPY src/wato_msgs/sample_msgs sample_msgs
 
 # Scan for rosdeps
 RUN apt-get -qq update && rosdep update && \
     rosdep install --from-paths . --ignore-src -r -s \
-        | grep 'apt-get install' \
-        | awk '{print $3}' \
-        | sort  > /tmp/colcon_install_list
+    | grep 'apt-get install' \
+    | awk '{print $3}' \
+    | sort  > /tmp/colcon_install_list
+
 
 ################################# Dependencies ################################
 FROM ${BASE_IMAGE} as dependencies
 
+RUN apt-get update && apt-get install -y \
+    git zip unzip libssl-dev libcairo2-dev lsb-release libgoogle-glog-dev libgflags-dev libatlas-base-dev libeigen3-dev software-properties-common \
+    build-essential cmake pkg-config libapr1-dev autoconf automake libtool curl libc6 libboost-all-dev debconf libomp5 libstdc++6 \
+    libqt5core5a libqt5xml5 libqt5gui5 libqt5widgets5 libqt5concurrent5 libqt5opengl5 libcap2 libusb-1.0-0 libatk-adaptor neovim \
+    python3-pip python3-setuptools \
+    && apt-get -qq autoremove -y && apt-get -qq autoclean && apt-get -qq clean && \
+    rm -rf /var/lib/apt/lists/* /root/* /root/.ros /tmp/* /usr/share/doc
+
+################################ INSTALL OpenCV with CUDA Support ##############
+WORKDIR /opt
+RUN git clone https://github.com/opencv/opencv.git && \
+    cd opencv && git checkout 4.5.5
+RUN git clone https://github.com/opencv/opencv_contrib.git && \
```

Comment:
directly clone from branch using `git clone -b`

Reply:
also put clones in the source section

Reply:
bet